ansible/custom_roles/mysql/tasks/configure.yml

```diff
@@ -15,6 +15,16 @@
     force: "{{ overwrite_global_mycnf }}"
   notify: restart mysql
 
+- name: Change datadir permissions to 777
```

Comment:
This is so confusing to me why it's needed... that said, yeah sure, we can deal with that later.

Reply:
Can you just add a comment explaining what happened and what we've tried?

Reply:
Yeah sure, I'll add a comment in the code.